applications/launchpad/gui-react/src/components/TBot/TBotPrompt/styles.ts

```diff
@@ -24,7 +24,7 @@ export const PromptContainer = styled(animated.div)<{ $floating?: boolean }>`
   position: ${({ $floating }) => ($floating ? 'fixed' : 'static')};
   right: 40px;
   bottom: 40px;
-  z-index: 1;
+  z-index: 200;
```

Comment:
It breaks the onboarding:

![image](https://user-images.githubusercontent.com/11715931/176165961-35c1d4d1-505b-47a2-b81e-d7237822b35c.png)

So maybe we need to change the TitleBar z-index as well, or make the `z-index` here dependent on `$floating`, ie:

```
z-index: ${({ $floating ) => $floating ? 200 : 1};
```

Reply:
Good catch. I'll check out the `$floating` approach.

Reply:
Fixed.